src/components/ui/sidebar/facility-nav.tsx

```diff
@@ -62,6 +65,14 @@ function generateFacilityLinks(
         {
           name: t("encounters"),
           url: `${baseUrl}/encounters/patients`,
+          children: [
+            patientId && encounterId
+              ? {
+                  name: t("encounter"),
+                  url: `${baseUrl}/patient/${patientId}/encounter/${encounterId}/updates`,
```

Comment:
```suggestion
                  url: `${baseUrl}/patient/${patientId}/encounter/${encounterId}`,
```

updates is just one of the tabs under encounter